ansible/roles/packages/tasks/debian.yaml

```diff
@@ -1,81 +1,68 @@
 ---
-  - name: install apt-transport-https remote deb package
-    apt:
-      name: apt-transport-https
-      state: latest
-      update_cache: yes
-    register: result
-    until: result is success
-    retries: 3
-    delay: 3
+- name: install apt-transport-https remote deb package
+  apt:
+    name: apt-transport-https
+    state: latest
+    update_cache: true
+  register: result
+  until: result is success
+  retries: 3
+  delay: 3
 
 
-  - name: apt update package management cache
-    apt:
-      update_cache: yes
-    register: result
-    until: result is success
-    retries: 3
-    delay: 3
+- name: apt update package management cache
+  apt:
+    update_cache: true
+  register: result
+  until: result is success
+  retries: 3
+  delay: 3
 
-  - name: install chrony remote deb package
-    apt:
-      name: chrony
-      state: present
-    register: result
-    until: result is success
-    retries: 3
-    delay: 3
+- name: install common debs
+  apt:
+    name:
+      - chrony
+      - libseccomp2
+      - nfs-common
+      - python3-cryptography
+      - python3-pip
```

Comment:
I consolidated the different sections into just this one like we do for RPM distros